client/blocks/conversation-caterpillar/docs/example.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { ConversationCaterpillar } from 'blocks/conversation-caterpillar';
+import { posts } from 'blocks/reader-post-card/docs/fixtures';
+
+const ConversationCaterpillarExample = () => {
+	const commentsTree = {
```

Comment:
do we have any commentsTree fixtures elsewhere? 
if not maybe we should make sure reusable commentsTree example data. 
🤔 

don't need to slow this pr up for that though